### DISCUSSION TITLE
makedepends and depends

### DISCUSSION THREAD

**Discussion by yuioto** [xfangfang/wiliwili#71] [Q&A]
# 不明确的depends要求

## 需求
- 只有编译时需要的
- 只有运行时需要的
- 都需要的

### makedepends
- git
- cmake
- gcc

> openssl由git提供。

### depends
- mpv

> ffmpeg由mpv提供。

### 不清楚是哪种depends
- python

> build的时候看到了python的字样

- glfw

我正在为Arch Linux制作PKGBUILD.

**Comment by xfangfang** [comment]
非常感谢您的贡献！

外部依赖有 mpv curl 和 某个tls库（mbedtls gnutls openssl都可以）

默认的cmake构建指令会自动下载curl，也可以通过 cmake选项 调整为使用系统库中的curl（不推荐，过新的curl会存在问题）

python 可能是mpv的依赖

项目内使用了修改后的glfw，不需要外部依赖。

**Reply by yuioto** [reply]
谢谢。它现在在我的电脑上正常工作。

我所写的PKGBUILD在这，希望它对一些人有用。
https://aur.archlinux.org/packages/wiliwili-git

**Reply by xfangfang** [reply]
> 谢谢。它现在在我的电脑上正常工作。
> 
> 我所写的PKGBUILD在这，希望它对一些人有用。 https://aur.archlinux.org/packages/wiliwili-git

非常感谢，我有空时候会更新一下readme把相关信息添加进来！
